lib/gladiator.rb

```diff
@@ -1,2 +1,19 @@
-class Gladiator
+#require "pry"
+
+class Gladiator# < ActiveRecord::Base
+
+  #belongs_to :arena
+
+  attr_accessor :name, :weapon
+
+  def initialize(name, weapon)
+    @name = name
+    @weapon = weapon
+  end
+
 end
+
+maximus = Gladiator.new("Maximus", "Trident")
```

Comment:
Move this usage code to your specs or a sandbox.rb.  Leave the Class files as just the Class definition.